DoSomethingArchive/legacy-website
# Prevents referer leakage in dsrealtime request

#### What's this PR do?

Adds a meta tag to the head which tells the browser / ajax request to not include the referrer in the headers.
It then removes the meta tag after the jax request is complete. 
#### How should this be reviewed?
1. configure the lobby url `admin/config/dosomething/dosomething_settings`
2. add a console log for the given url to check the headers & see if refer is present

```
11:18:30 AM web.1 |    headers: 
11:18:30 AM web.1 |     { host: '4361ebf3.ngrok.io',
11:18:30 AM web.1 |       'content-length': '7',
11:18:30 AM web.1 |       'cache-control': 'max-age=0',
11:18:30 AM web.1 |       accept: '*/*',
11:18:30 AM web.1 |       origin: 'http://dev.dosomething.org:8888',
11:18:30 AM web.1 |       'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36',
11:18:30 AM web.1 |       'content-type': 'application/x-www-form-urlencoded; charset=UTF-8',
11:18:30 AM web.1 |       'accept-encoding': 'gzip, deflate',
11:18:30 AM web.1 |       'accept-language': 'en-US,en;q=0.8',
11:18:30 AM web.1 |       'x-fastly-city': 'New York',
11:18:30 AM web.1 |       'x-fastly-country-code': 'US',
11:18:30 AM web.1 |       'x-fastly-country-name': 'United States',
11:18:30 AM web.1 |       'x-fastly-latitude': '40.739.',
11:18:30 AM web.1 |       'x-fastly-longitude': '-73.983',
11:18:30 AM web.1 |       'x-fastly-region': 'NY',
11:18:30 AM web.1 |       'x-forwarded-for': '207.110.19.130' },
```

If you want to test how the meta tag appears in the DOM, remove the `.remove()' at the end of the function.
#### Any background context you want to provide?

This is currently able to expose sensitive information. See: https://cobalt.io/dosomething/dosomething/reports/93#comment_39620
#### Relevant tickets

Fixes https://github.com/DoSomething/devops/issues/152
#### Checklist
